docs/external_api.md

```diff
@@ -0,0 +1,1526 @@
+## Params
+
+```per_page=10  # default 10```
+
+```page=5       # default 1```
+
+```updated_after=Thu, 26 May 2022 14:35:09 +0100       # default nil```
```

Comment:
Wowzers, that's some very human readable parameterising. Is it worth specifying a few different formats you can provide this in though, for instance if someone just wanted to look from a particular date without checking what day of the week that date was? Seems like it works if you just cut out the day, so maybe you could give a few examples of more or less specific ways of specifying the date

Reply:
Updated

Reply:
Not seeing the update. I think it was meant to be in [this commit](https://github.com/unepwcmc/action-agenda/pull/110/commits/5f95d50a2be6a81f8f298735540fa5b1843d241e)?